home/tests/test_models.py

```diff
@@ -495,6 +496,35 @@ def test_status_live_plus_draft(self):
         ordered_content_set.save_revision()
         self.assertEqual(ordered_content_set.status(), "Live + Draft")
 
+    def test_get_relationship(self):
+        ordered_content_set = OrderedContentSet(name="Test Title")
```

Comment:
Can you add explanatory docstrings to all these tests, please? (Look at some of the existing tests (particularly around import/export and views) for examples.)

Reply:
Good idea, I've added them.

Reply:
My bad, fixed.